### PR TITLE
animate settings visibility

### PR DESCRIPTION
**Pull request by oetiker**
this makes the settings window animated

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "files.insertFinalNewline": true,
     "files.trimFinalNewlines": true,
     "files.trimTrailingWhitespace": true,
-    "editor.tabSize": 4,
+    "editor.tabSize": 2,
```
Comment (pcrespov) [inline .vscode/settings.json]: This is fine with me but I would then recommend to set python linter to 2 spaces/tab as well ... this way everything is coherent

File: services/web-client/source/class/qxapp/desktop/PrjEditor.js [hunk 2]
```diff
@@ -30,32 +48,39 @@ qx.Class.define("qxapp.desktop.PrjEditor", {
...
+
+    this.__TransDeco = new qx.ui.decoration.Decorator().set({
+      transitionProperty: ["left", "right", "width"],
+      transitionDuration: "0.1s",
+      transitionTimingFunction: "ease"
+    });
   },
 
   members: {
     __Pane: null,
```
Comment (pcrespov) [inline services/web-client/source/class/qxapp/desktop/PrjEditor.js]: didn't we agree on naming variables as ``__myVariable``? I wonder why this is not captured by the lintier? Please tune the ESlinter to catch all these conventions.

File: .vscode/settings.json (hunk 1 above)
Comment (oetiker) [reply .vscode/settings.json]: I personally would prefer 4 ... but having 2 in the linter and 4 in vscode is not ideal

File: services/web-client/source/class/qxapp/desktop/PrjEditor.js (hunk 2 above)
Comment (oetiker) [reply services/web-client/source/class/qxapp/desktop/PrjEditor.js]: yes by all means! __xyzAbc seems much nicer to me ...

**Pull request closed without merging**